spec/c02controlflow/validate/balanced_parenthesesSpec.js

```diff
@@ -0,0 +1,36 @@
+const balancedParentheses = require('../../../src/c02controlflow/validate/balanced_parentheses')
+describe('Balances parentheses', function () {
+
+    it('Should return false when string are not balanced', function () {
```

Comment:
Use `parentheses` instead of `string`.
`Should return false when parentheses are not balanced`
Same for all tests